src/adapters/kafka/index.ts

```diff
@@ -14,16 +15,32 @@ class KafkaAdapter extends Adapter {
     this.kafka = new Kafka({
       clientId: 'glee-app',  // clientID: hardcoded need to change afterwards 
       brokers: [brokerUrl.host],
-      ssl: true,
+      ssl: {
```

Comment:
i am just testing stuff and trying to putting it over here, also in SSL we need all these properties right?
Also how do would we get the values that needed to be passed like `client.key, client.pem`